tests/functional/object/mcg/test_block_external_access.py

```
@@ -367,8 +416,8 @@ def test_block_access_from_noobaa(
 
         self.check_disable_routes(False)
 
-        self.set_disable_routes_value(noobaa_obj, False, True)
+        self.set_disable_routes_value(storagecluster_obj, True, True)
```

Ruff (0.15.21)

[warning] Boolean positional value in function call

(FBT003)